Doc/library/struct.rst

```diff
@@ -279,9 +279,8 @@ Notes:
 (1)
    .. index:: single: ? (question mark); in struct format strings
 
-   The ``'?'`` conversion code corresponds to the :c:expr:`_Bool` type defined by
-   C99. If this type is not available, it is simulated using a :c:expr:`char`. In
-   standard mode, it is always represented by one byte.
+   The ``'?'`` conversion code corresponds to the :c:expr:`_Bool` type
+   defined by C standards since C99.
```

Comment:
Keep a note about standard mode.

Reply:
Thanks, I misinterpret that part, thinking that it's about default mode.